### DISCUSSION TITLE
Hello world test failing ,receiving vusers.failed and errors.enotfound error message

### DISCUSSION THREAD

**Discussion by rrnareshkumar** [artilleryio/artillery#1996] [Artillery]
Hi,

Executed hello world test after successfully installing artillery . Hello world test failing ,receiving vusers.failed and errors.enotfound error message((refer the error snapshot). Please find the other details below , kindly share your suggestions to fix this issue .

Error Message:

Test run id: tthax_t8j8apfw8d7bmqzfhyqry3fnjwad6_aygz
Phase started: unnamed (index: 0, duration: 2s) 01:13:29(-0400)

Phase completed: unnamed (index: 0, duration: 2s) 01:13:31(-0400)

--------------------------------------
Metrics for period to: 01:13:40(-0400) (width: 1.066s)
--------------------------------------

errors.ENOTFOUND: .............................................................. 4
http.request_rate: ............................................................. 4/sec
http.requests: ................................................................. 4
vusers.created: ................................................................ 4
vusers.created_by_name.0: ...................................................... 4
vusers.failed: ................................................................. 4

Warning: multiple batches of metrics for period 1689657210000 2023-07-18T05:13:30.000Z
All VUs finished. Total time: 2 seconds

--------------------------------
Summary report @ 01:13:32(-0400)
--------------------------------

errors.ENOTFOUND: .............................................................. 4
http.request_rate: ............................................................. 4/sec
http.requests: ................................................................. 4
vusers.created: ................................................................ 4
vusers.created_by_name.0: ...................................................... 4
vusers.failed: ................................................................. 4

Below is the yaml code which I used. 

config:
  target: https://artillery.io
  phases:
    - duration: 2
      arrivalRate: 2
scenarios:
  - flow:
      - get:
          url: /docs

Also I ran a test using below quick command but receiving the receiving vusers.failed and errors.enotfound error message (refer the error snapshot) . Please note that I tried with different apps or URL's and getting the same error message .

artillery quick --count 2 -n 4 https://artillery.io/

**Comment by hassy** [comment]
`ENOTFOUND` is a DNS resolution error. It sounds like a local networking issue if you're seeing it when testing different apps/URLs.

**Reply by rrnareshkumar** [reply]
thank you . It seems it was related CSP / blocked by corporate proxy . I was able to test API/Application which was hosted internally .
